tests/e2e/test_spyre_cb.py

```diff
@@ -114,35 +114,52 @@ def get_params_test_blocks_borders_aligned_prompts():
     seqs_max_tokens = [65, 67, 7]
     prompts_lengths = [49, 41, 47]
     steps_add_reqs = [0, 0, 0]  # add all requests in the beginning
+    available_blocks = -1  # no restriction
+    max_num_seqs = 2
 
     checked_steps = [
         {
             "step": 0,
             "tkv": 0,
             "waiting": ["0", "1", "2"],
             "running": [],
-            "request_outputs": []
+            "request_outputs": [],
+            "n_reserved_blocks": 0,
```

Comment:
For the future, but I am wondering if it is necessary to always add newly tested values to all the previous existing tests, this can quickly become cumbersome to maintain. Maybe in the main running loop we could tests the keys values only if they are found in the reference dictionary, this way we can only add target tested attributes in the corresponding test

Reply:
ah absolutely, as I said it is a comment for the future. I didn't mean to remove the existing `request_outputs` or add more logic 👍

Reply:
For the future we could also consider converting these to snapshot tests. The idea there is that for any new test cases you only need to specify the inputs, and then the test can snapshot the outputs automatically for your review. The snapshots are stored as data files read by the test, and when schema changes like this happen it's simple to just regenerate all the snapshots